src/rocketData.py

```diff
@@ -226,84 +72,16 @@ def __init__(self):
                 float,
                 float,
                 float,
-                float
             ],
-            'encoders': self._encoders,
-            'time_stamp': float
+            'encoder': self._encoders,
+            'timestamp': float
         }
 
-    # @property decorator makes the function below a getter
+
     @property
     def data(self):
         return self.data
 
-    @property
-    def lsm(self):
-        return self.data['lsm']
-
-    @lsm.setter
-    def lsm(self, vals):
-        if len(vals) == 10:
-            ax, ay, az, mx, my, mz, gx, gy, gz, t = vals
-            self.data['lsm']['acceleration_x'] = ax
-            self.data['lsm']['acceleration_y'] = ay
-            self.data['lsm']['acceleration_z'] = az
-            self.data['lsm']['magnetometer_x'] = mx
-            self.data['lsm']['magnetometer_y'] = my
-            self.data['lsm']['magnetometer_z'] = mz
-            self.data['lsm']['gyroscope_x'] = gx
-            self.data['lsm']['gyroscope_y'] = gy
-            self.data['lsm']['gyroscope_z'] = gz
-            self.data['lsm']['temperature'] = t
-        else:
-            print("LSM accepts 10 values in this order: Acceleration (x, y, z), magnetometer (x, y, z), gyroscope (x, y, z)")
-
-    @lsm.setter
-    def lsm_temp(self, t):
-        self.data['lsm']['temperature'] = t
-
-    @property
-    def bme(self):
-        return self.data['bme']
-
-    @bme.setter
-    def bme(self, vals):
-        t, h, p = vals
-        if len(vals) == 3:
-            self.data['bme']['temperature'] = t
-            self.data['bme']['humidity'] = h
-            self.data['bme']['pressure'] = p
-        else:
-            print("BME accept exactly 3 values in this order: temperature, Humidity, Pressure")
-
-    @bme.setter
-    def bme_temp(self, t):
-        self.data['bme']['temperature'] = t
-
-    @property
-    def strain_gauges(self):
-        return self.data['strain_gauges']
-
-    @strain_gauges.setter
-    def strain_gauges(self, vals):
-        if len(vals) == 12:
-            self.data['strain_gauges'] = vals
-        else:
-            print("Strain gauges must be exactly 12 values")
-
-    @property
-    def encoders_position(self):
-        return self.data['encoders']['position']
-
-    @property
-    def encoders_percent(self):
-        return self.data['encoder']['percent']
-
-    @encoders.setter
-    def encoders(self, p):
-        pos, perc = p
-        self.data['encoders']['position'] = pos
-        self.data['encoders']['percent'] = perc
 
     @property
     def time_stamp(self):
```

Comment:
```suggestion
    def timestamp(self):
```

Reply:
Also the line below this where you have:
```python
return self.data['time_stamp']
```
This won't succeed because further up in the file you changed it to have no underscore